docs/src/tutorials/consistent_system/consistent_system.md

```diff
@@ -0,0 +1,50 @@
+# Solving a Consistent Linear System
+
+This guide demonstrates how to use `RLinearAlgebra.jl` package to solve a 
+**consistent linear system**---a system where at least one solution 
+exists---expressed in the form:
+
+$$Ax = b.$$
+
+We'll walk through setting up the problem, using a solver, and verifying the result.
+
+
+First, let's define our linear system $Ax = b$ with some known solution `x_true`.
+
+
+```julia
+using LinearAlgebra
+num_rows, num_cols = 1000, 50;
+A = randn(Float64, num_rows, num_cols);
+x_true = randn(Float64, num_cols);
+b = A * x_true;
```

Comment:
You can add a line here and make this an `@example` block so that the last line is `norm(A*x_true - b)`.